pytorch/training/docker/1.13/py3/cu117/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,118 @@
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.13.1-gpu-py39-cu117-ubuntu20.04-sagemaker AS sagemaker
+
+########
+###   :::Update NCCL:::
+###   This is a temporary workaround to
+###   upgrade NCCL
+########
+
+# Remove Older NCCL
+RUN rm -rvf `find /usr/local | grep nccl` /usr/local/obj
+
+ENV NCCL_VERSION=2.14.3
```

Comment:
Looks like the base container has the same NCCL version. Why is this reinstall necessary ?

Reply:
It's a good idea to get rid of the reinstalling, including sagemaker and its training toolkit. I think Bruce maintains the structure from the last release here. We should revamp and get rid of these in PT2.0

Reply:
We have some new feature in sagemaker training toolkit 4.4.5 like this: https://github.com/aws/sagemaker-training-toolkit/pull/171/files. Not sure whether we need to upgrade this.

Reply:
yes we will need upgrade. 
Sorry I forgot about this. I've added this upgraded in the base image revamp PR but haven't merged yet.

Reply:
Got it. Then I will remove this part in Hopper DLC.